src/index.html

```diff
@@ -22,6 +22,294 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
```

Comment:
Fix code style everywhere if you have only 2 attributes
```suggestion
      <a href="#" class="header__link">
```